src/CameraOnvif.hpp

```diff
@@ -0,0 +1,48 @@
+#ifndef CAMERA_ONVIF_CAMERAONVIF_HPP
+#define CAMERA_ONVIF_CAMERAONVIF_HPP
+
+#include <string>
+
+namespace camera_onvif {
+    /**
+     *
+     */
+    class CameraOnvif {
+        struct Private;
+        Private* m_private = nullptr;
+
+        std::string user;
+        std::string pass;
+        std::string uri;
```

Comment:
We always prefix member variables with m_ (as you did with m_private), this also makes the code easier to read because it makes it pretty obvious where the variables are coming from. Please, be consistent with the style

Reply:
Sure, I didn't noticed it, previously it was Sylvain suggestion of making this m_private.